src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,48 @@
+package calculator
+
+class Calculator {
+    companion object {
+        private val ArithmeticOperations = listOf<String>("+", "-", "*", "/")
+
```

Comment:
음 변수가 대문자로 시작하는데 Lint에 안걸리네.. 상수 자리여서그런가?
자바, 코틀린에서 절대 변하지 않을 상수 영역은 Snake case로 모두 대문자로 쓰는게 컨벤션이야.
```suggestion
        private val ARITHMETIC_OPERATIONS = listOf<String>("+", "-", "*", "/")

```